recipes/cni/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "cni"%}
+{% set version = "0.6.0"%}
+
+{% set sha256 = "802f4a002b4eb774624a9dc1c859d3c9926eb2d862e66a673fc99cfc8bcd7494" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/containernetworking/{{ name }}/archive/v{{ version }}.tar.gz  # [not win]
+  fn: {{ name }}-{{ version }}.tar.gz  # [not win]
+  sha256: {{ sha256 }}  # [not win]
+
+build:
+  skip: True  # [not linux]
+  number: 0
+
+requirements:
+  build:
+    - go 1.8.*  # [not win]
+    - make  # [not win]
+    - toolchain  # [not win]
+  run:
+    - jq
+
+test:
+  commands:
+    - which cnitool
```

Comment:
CNITool does not have a --help or --version, and its other calls require root level access.